libs/core/src/components/sage-tooltip/stories/sage-tooltip.docs.stories.mdx

```diff
@@ -0,0 +1,84 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+import { html, render } from 'lit-html';
+
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+
+import * as stories from './sage-tooltip.stories';
+
+<Meta
+  title="components/Tooltip"
+  component="sage-tooltip"
+  argTypes={extractArgTypes('sage-tooltip')}
+/>
+
+# Tooltip
+<p>Tooltips display helpful popup text when users hover over an element.</p>
+
+## Guidelines
+
+## Properties
+
+<ArgsTable of="sage-tooltip" />
+
+## Variants
+### Trigger
+Whether your trigger is text or an HTML element, it will be place in the default slot between the opening and closing tags.
```

Comment:
"...be place" (place has a typo) and do we need to mention the default slot? indicates to me there may be other non-default slots.

Reply:
Yes, we need to mention the default slot since there will only be one. You can have other "named" slots, but only one default slot